cosmos/providers/dbt/core/profiles/tests/profiles_test.py

```diff
@@ -211,7 +203,7 @@ def test_create_profile_vars_postgres(airflow_connection: Connection) -> None:
         "POSTGRES_HOST": host,
         "POSTGRES_USER": login,
         "POSTGRES_PASSWORD": password,
-        "POSTGRES_DATABASE": schema,
+        "POSTGRES_DATABASE": database,
```

Comment:
Currently, there is an issue in the Airflow Postgres Hook where schema references the database. Was this change accidental?

Reply:
good catch, I added a test specifically for that here: https://github.com/astronomer/astronomer-cosmos/pull/252/commits/64d99ba10af26e5b25a8acce507b2936fe8c1586